<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgetsClass</class>
 <widget class="QWidget" name="QtWidgetsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>816</width>
    <height>599</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>200000</width>
    <height>500000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QtWidgetsClass</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0">
    <widget class="QPushButton" name="pushButton_2">
     <property name="maximumSize">
      <size>
       <width>93</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>完成</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="pushButton_8">
     <property name="maximumSize">
      <size>
       <width>93</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>完成</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QTextEdit" name="textEdit">
     <property name="minimumSize">
      <size>
       <width>500</width>
       <height>100</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>200000</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QPushButton" name="pushButton_7">
     <property name="maximumSize">
      <size>
       <width>93</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>完成</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_4">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_1">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="pushButton_1">
     <property name="maximumSize">
      <size>
       <width>93</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>完成</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="pushButton_5">
     <property name="maximumSize">
      <size>
       <width>93</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>完成</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>添加任务</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_3">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pushButton_3">
     <property name="maximumSize">
      <size>
       <width>93</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>完成</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label_2">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButton_4">
     <property name="maximumSize">
      <size>
       <width>93</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>完成</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_5">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="label_8">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="pushButton_6">
     <property name="maximumSize">
      <size>
       <width>93</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>完成</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label_6">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_7">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>50</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>50</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_1</sender>
   <signal>clicked()</signal>
   <receiver>label_1</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>37</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>label_2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>98</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>93</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>label_3</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>149</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>label_4</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>210</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>label_5</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>83</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>186</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>label_6</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>190</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>label_7</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>85</x>
     <y>375</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>label_8</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>167</x>
     <y>438</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_1</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsClass</receiver>
   <slot>check()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsClass</receiver>
   <slot>check()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>15</x>
     <y>94</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_3</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsClass</receiver>
   <slot>check()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsClass</receiver>
   <slot>check()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>17</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>5</x>
     <y>211</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsClass</receiver>
   <slot>check()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>18</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>6</x>
     <y>268</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_6</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsClass</receiver>
   <slot>check()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>21</x>
     <y>320</y>
    </hint>
    <hint type="destinationlabel">
     <x>3</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsClass</receiver>
   <slot>check()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>29</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>4</x>
     <y>379</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_8</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsClass</receiver>
   <slot>check()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgetsClass</receiver>
   <slot>add()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>507</y>
    </hint>
    <hint type="destinationlabel">
     <x>48</x>
     <y>466</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>add()</slot>
  <slot>check()</slot>
 </slots>
</ui>
